packages/upward-js/package.json

```diff
@@ -38,14 +38,15 @@
     "debug": "~4.1.1",
     "debug-error-middleware": "~1.3.0",
     "dotenv": "~6.2.0",
-    "graphql": "~14.1.1",
+    "graphql": "~14.2.1",
     "graphql-tag": "~2.10.1",
     "hogan.js": "~3.0.2",
     "http-proxy-middleware": "~0.19.1",
     "js-yaml": "~3.12.1",
     "lodash": "~4.17.11",
     "morgan": "~1.9.1",
-    "node-fetch": "~2.3.0"
+    "node-fetch": "~2.3.0",
+    "shrink-ray-current": "~4.0.0"
```

Comment:
This re-introduces the bug solved by #864, this can't be a hard dependency.

Reply:
[It can now](https://github.com/Alorel/shrink-ray/releases/tag/4.0.0)! Instead of having transitive dependencies on native bindings, we can now have direct dependencies. That's always better. So that's what the new optionalDependencies are!

Reply:
Neat! I thought the offending dependency was `node-gyp`, but it appears it's not a direct dependency of `shrink-ray-current`. Can we confirm this is working as expected on a machine w/o Python?

Reply:
I don't think the container has Python, but I've tested with a local container lacking python as well and it doesn't choke there.